addons/hr/static/tests/web/m2x_avatar_user.test.js

```diff
@@ -71,9 +58,10 @@ test("avatar card preview with hr", async () => {
         </kanban>`,
     });
     await contains(".o_m2o_avatar > img").click();
-    expect.verifySteps(["user read"]);
-    expect(".o_avatar_card").toHaveCount(1);
-    expect(".o_avatar_card span[data-tooltip='Work Location'] .fa-building-o").toHaveCount(1);
+    await mailContains(".o_avatar_card", { count: 1 });
+    await mailContains(".o_avatar_card span[data-tooltip='Work Location'] .fa-building-o", {
+        count: 1,
+    });
```

Comment:
`count: 1` can be omitted, this is implicitly the value and it crashes when different from `1` (0 or strictly more than `1`).